src/components/content-tab-overview.tsx

```diff
@@ -142,18 +140,10 @@ function CustomizeSection( {
 
 function ShortcutsSection( { selectedSite }: Pick< ContentTabOverviewProps, 'selectedSite' > ) {
 	const { terminalWpCliEnabled } = useFeatureFlags();
-	const { terminal, handleTerminalChange, getSavedTerminal } = useTerminalData();
-	const { editor, handleEditorChange, getSavedEditor } = useEditorData();
+	const { data: editor } = useGetUserEditorQuery();
+	const { data: terminal } = useGetUserTerminalQuery();
 
-	useEffect( () => {
-		handleEditorChange( editor );
-		handleTerminalChange( terminal );
-	}, [ editor, handleEditorChange, handleTerminalChange, terminal ] );
-
-	useIpcListener( 'user-preference-changed', async () => {
-		handleEditorChange( await getSavedEditor() );
-		handleTerminalChange( await getSavedTerminal() );
-	} );
```

Comment:
Did we rely on this logic before, or was it just a kind of nice to have?

Reply:
We did previously, this was introduced when we updated the settings screen: https://github.com/Automattic/studio/pull/1215